### ISSUE TITLE
Don't understand syntax here

### ISSUE THREAD

**Issue by MrBojangles** [zalmoxisus/redux-devtools-extension#579]
TLDR: I want to know what a red number represents as an object type.
I'm having an issue with updating my state with React/Redux. I send a form. Get a response from server then part of my state object changes in a weird way. see pictures below
before i have submitted form
<img width="304" alt="screenshot 2018-11-19 at 09 58 16" src="https://user-images.githubusercontent.com/33279255/48699790-363e7800-ebe2-11e8-9c89-c95dfc1b32cc.png">
afterwards i cant access the images array cuz its a number now???....
<img width="349" alt="screenshot 2018-11-19 at 09 58 39" src="https://user-images.githubusercontent.com/33279255/48699819-47878480-ebe2-11e8-8ac4-ef4b7f9f0593.png">

**Comment by ZachariahTheMagnificent** [comment]
What a noob lol

**Comment by MrBojangles** [comment]
am posting images
```
export const uploadImage = (formProps) => async dispatch => {
  const formData = new FormData();
  formData.append('imageUpload', formProps.item('0'));
  const user = localStorage.getItem('currentUser');
  const token = localStorage.getItem('token');
  const options = {
    method: 'POST',
    headers: { 'content-type': 'multipart/form-data',
      authorization: token},
    data: formData,
    url: `http://localhost:3000/api/users/${user}/images`
  };
  try {
    const response = await axios(options);
    dispatch({ type: UPLOAD_IMAGE, payload: response.data });
  } catch(e) {
    dispatch({ type: IMAGE_ERROR, payload: e});
  }
};
```
